.travis.yml

```diff
@@ -17,4 +17,13 @@ notifications:
     recipients:
       - ajax@bynder.com
     on_success: never
-    on_failure: always
\ No newline at end of file
+    on_failure: always
+
+deploy:
+  provider: pypi
+  username: __token__
+  password:
+    secure: $PYPI_TOKEN
+  on:
+    tags: true
+    repo: Bynder/bynder-python-sdk
```

Comment:
Is it necessary to specify repo?

Reply:
It is not necessary. I was modelling it off the docs where most of them had this. This is useful when we want to public off different forks. I am presuming we don't do that ?

Reply:
Correct, we don't do that.